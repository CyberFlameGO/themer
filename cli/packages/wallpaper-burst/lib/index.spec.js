const { render, renderInstructions } = require('./index');
const { colors } = require('../../colors-default');

describe('themer "burst" wallpaper', () => {
  it('should not contain "undefined" in the SVG output', async () => {
    const files = await Promise.all(render(colors, {}));
    files.forEach(file => {
      expect(file.contents.toString('utf8')).not.toContain('undefined');
    });
  });
  it('should list output files', async () => {
    const files = await Promise.all(render(colors, { 'themer-wallpaper-diamonds-size': '1000x1000' }));
    const instructions = renderInstructions(files.map(({ name }) => name));
    expect(instructions).toMatchSnapshot();
  });
});
