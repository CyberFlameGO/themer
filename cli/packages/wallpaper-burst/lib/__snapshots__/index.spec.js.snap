// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`themer "burst" wallpaper should list output files 1`] = `
"
Files generated:

* \`themer-wallpaper-burst-dark-2880x1800.svg\`
* \`themer-wallpaper-burst-light-2880x1800.svg\`
* \`themer-wallpaper-burst-dark-750x1334.svg\`
* \`themer-wallpaper-burst-light-750x1334.svg\`
"
`;
